.storybook/config.js

```diff
@@ -0,0 +1,11 @@
+const {configure} = require('@kadira/storybook');
+
+require("../data/content/main.css");
```

Comment:
This might actually be better as a static directory (it would reduce the additional configuration needed in webpack) `https://github.com/kadirahq/react-storybook/blob/master/docs/configure_storybook.md#static-directory`